yarn-project/Earthfile

```diff
@@ -49,9 +52,8 @@ build:
 
     WORKDIR /usr/src/yarn-project
     COPY . .
-    ENV PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true
     RUN ./bootstrap.sh full
-    RUN cd ivc-integration && chmod +x run_browser_tests.sh && npx playwright install && npx playwright install-deps
+    RUN cd ivc-integration && chmod +x run_browser_tests.sh
```

Comment:
Also I'm not sure why we haven't just chmodded `run_browser_tests.sh` and commit it with the correct permissions.